packages/client/foreman_ygg_worker/foreman_ygg_worker.spec

```diff
@@ -34,7 +38,12 @@ Url: https://github.com/%{repo_orgname}/%{repo_name}/
 %endif
 ExclusiveArch: %{go_arches}
 
+%if 0%{?suse_version}
+BuildRequires: go
+%else
 BuildRequires: golang
+BuildRequires: redhat-rpm-config
```

Comment:
This is an unusual one. I think this is expected to be in the buildroot already. Can you explain why you added this?

Reply:
I have added this to make it more explicit where the hardened linker on EL comes from. Note that this is not available on OpenSUSE/SLES.

From my commit message:

> Require redhat-rpm-config for go
>   When building the Foreman Client for REX pull mode, you have to compile
>   go code. On EL, you can set a flag that requires the file
>   "redhat-hardened-ld" which is part of the "redhat-rpm-config" package.
>   See https://fedora.pkgs.org/rawhide/fedora-x86_64/redhat-rpm-config-264-1.fc40.noarch.rpm.html


I am not sure what you mean by "buildroot". If it's always present when building RPMs for/on EL using foreman-packaging, then I can revert this.